cpu/stm32_common/periph/flash_common.c

```diff
@@ -67,6 +67,7 @@ void _wait_for_pending_operations(void)
 
     /* Clear 'end of operation' bit in status register */
     if (FLASH->SR & FLASH_SR_EOP) {
-        FLASH->SR &= ~(FLASH_SR_EOP);
+        FLASH->SR |= FLASH_SR_EOP;
```

Comment:
Maybe we could just remove the if line above and just keep this line?

Reply:
We could do this and guard it for STM32F2, STM32F4, STM32F7 & STM32L4 since EOPIE is not set this is not needed for these platforms, a proper commit comment would justify and documment in case in a future EOPIE are used. What do you think?